webpack/components/AnsibleHostDetail/components/AnsibleHostInventory/index.js

```diff
@@ -0,0 +1,20 @@
+import React, { useMemo } from 'react';
+import { foremanUrl } from 'foremanReact/common/helpers';
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import PropTypes from 'prop-types';
+import AnsibleHostInventory from './AnsibleHostInventory';
+
+const WrappedAnsibleHostInventory = ({ hostId }) => {
+  const params = useMemo(() => ({ params: { host_ids: [hostId] } }), [hostId]);
```

Comment:
Just out of curiosity, why did you decide to `useMemo`?

Reply:
That was @amirfefer input. I think because we want it to change when the hostId changes but Amir if there is a better explanation let us know :)